assembl/static2/package.json

```diff
@@ -9,8 +9,8 @@
   "scripts": {
     "test": "jest",
     "test:watch": "npm test -- --watch",
-    "dev": "webpack -d --watch",
-    "build": "webpack -p",
+    "dev": "webpack -d --watch --progress --colors",
```

Comment:
I do not seem to have --progress in my version of webpack installed locally. Also, documentation doesn't seem to have --colors.